src/BlockEntities/MobSpawnerEntity.cpp

```diff
@@ -260,7 +269,7 @@ int cMobSpawnerEntity::GetNearbyMonsterNum(eMonsterType a_EntityType)
 				return;
 			}
 
-			if ((Diff(m_SpawnerPos.x, a_Entity->GetPosX()) <= 8.0) && (Diff(m_SpawnerPos.y, a_Entity->GetPosY()) <= 4.0) && (Diff(m_SpawnerPos.z, a_Entity->GetPosZ()) <= 8.0))
+			if ((Diff(m_SpawnerPos.x, a_Entity->GetPosX()) <= (2 * m_SpawnRange + 1)) && (Diff(m_SpawnerPos.y, a_Entity->GetPosY()) <= 4.0) && (Diff(m_SpawnerPos.z, a_Entity->GetPosZ()) <= (2 * m_SpawnRange + 1)))
```

Comment:
The wiki says
>Overrides the maximum number of nearby (within a box of spawnrange*2+1 × spawnrange*2+1 × 8 centered around the spawner block) entities whose IDs match this spawner's entity ID.

The radius of that box should be half the width, so `spawnrange + 0.5`.

Reply:
Really, this whole function should be reworked to use cWorld::ForEachEntityInBox

Reply:
This might resolve the lag that is introduced when spawning a ton of spanners

Reply:
It does - yeah 🤯